### DISCUSSION TITLE
【转载】假小子圣经

### DISCUSSION THREAD

**Discussion by JustOldJohn** [share121/inter-knot#1664] [其他]
早年在4chan上发源，现作为圣经传颂
# 假门 🙏
![屏幕截图 2024-08-12 141255](https://github.com/user-attachments/assets/27137c6c-3286-4bb8-ab6a-659aa30ab6e3)
![屏幕截图 2024-08-12 141331](https://github.com/user-attachments/assets/052cd211-ccc2-46db-8b4a-b53ed78a758b)
![屏幕截图 2024-08-12 141355](https://github.com/user-attachments/assets/e535e3bb-24ec-4669-9be3-4eaa13b79930)
![屏幕截图 2024-08-12 141415](https://github.com/user-attachments/assets/3f39dff7-ba67-48ac-a5e4-b73f71b258d9)
![屏幕截图 2024-08-12 141434](https://github.com/user-attachments/assets/32eeb628-f4ca-4af3-b74b-b99b8661ec33)
![屏幕截图 2024-08-12 141455](https://github.com/user-attachments/assets/297b7511-a1bb-47fb-87e6-d201e4f8057f)
![屏幕截图 2024-08-12 141513](https://github.com/user-attachments/assets/d1278fd0-81ab-4528-acda-c17f9f03bab7)

**Comment by hzjwork** [comment]
总有一天所有人都会控中性风格的.jpg

**Reply by JustOldJohn** [reply]
>I LIKE THOSE WOMEN THAT CAN KICK MY ASS.jpg

**Comment by BaCktOZwero** [comment]
bro，既然这样为何不加入男娘门派（合十）

**Reply by JustOldJohn** [reply]
> bro，既然这样为何不加入男娘门派（合十）

假小子与男娘是无法覆盖对方的，男娘是强调女性特征的男生，假小子是抛弃了女性刻板印象，以非甜妹的姿态自由展现自己的女生！
# **假门🙏**

**Comment by lllliufeng** [comment]
我看看sunoai怎么说（
